gybitg/SeedData.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections.Generic;
+using Bogus;
+using System.Linq;
+using System.Threading.Tasks;
+using gybitg.Data;
+using gybitg.Models;
+using gybitg.Models.ManageViewModels;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace gybitg
+{
+    public class SeedData
+    {
+        public static async Task InitializeAsync(IServiceProvider services)
+        {
+            await Seed(services.GetRequiredService<ApplicationDbContext>());
+        }
+
+        public static async Task Seed(ApplicationDbContext context)
+        {
+            if (context.AthleteUserViewModel.Any())
+            {
+                return; //already has data, don't add any more test data
+            }
+            if (context.AthleteStats.Any())
+            {
+                return;
+            }
+
+            // Trying to get data for the db context for testing
+            //this was not working correctly
+            //var firstName = new[] { "Adam", "Alex", "Kevin", "Daniel", "Keith" };
```

Comment:
Take a look at http://jackhiston.com/2017/10/1/how-to-create-bogus-data-in-c/ for some help with this when needed.

Reply:
Great post/info.  Thanks, Brian!

Reply:
Yep, my next focus has been the local repo for the four athletes.